src/twisted/python/_pydoctortemplates/subheader.html

```diff
@@ -5,25 +5,5 @@
     </a>
   </div>
 
-  <!-- Google analytics, obviously. -->
-  <script src="//www.google-analytics.com/urchin.js" type="text/javascript"></script>
-  <script type="text/javascript">
-    _uacct = "UA-99018-6";
-    urchinTracker();
-  </script>
-
-  <!-- If the documentation isn't current, insert a current link. -->
-  <script type="text/javascript">
-    if (window.location.pathname.indexOf('/current/') == -1) {
-      <!-- Give the user a link to this page, but in the current version of the docs. -->
-      var link = document.getElementById('current-docs-link');
-      link.href = window.location.pathname.replace(/\/\d+\.\d+\.\d+\/api\//, '/current/api/');
-      <!-- And make it visible -->
-      var container = document.getElementById('current-docs-container');
-      container.style.display = "";
-      delete link;
-      delete container;
-    }
-  </script>
-  
+  <script src="stable-link.js" type="text/javascript"></script>
```

Comment:
Happy to see the google analytics removed.

-------

Just asking 

Does moving the script to a separate file improves the performance ?